.drone.yml

```diff
@@ -35,7 +35,9 @@ x-anchors:
     - sleep 20
 
   check-format: &check-format
-    - docker run --rm -v $(pwd):/data -w /data hashicorp/terraform fmt --diff --check
+    - terraform --version
+    - terraform fmt --diff --check
+#    - docker run --rm -v $(pwd):/data -w /data hashicorp/terraform fmt --diff --check
```

Comment:
It is better to remove this completely, rather than comment it out.

Reply:
Removed